apache/trafficcontrol
# Add TO validation to ensure a parent.config weight parameter is a float

## What does this PR (Pull Request) do?


- [x] This PR fixes #3514 by adding validation to ensure a parent.config weight parameter is a float

## Which Traffic Control components are affected by this PR?


- Documentation
- Traffic Ops

## What is the best way to verify this PR?

* Add a parameter
    - named `weight`
    - with config file `parent.config`
    - with a valid float value

* Make sure you cannot create or update a parameter to be
  - named `weight`
  - with config file `parent.config`
  - have a value that does not parse to a valid float

## The following criteria are ALL met by this PR




## Additional Information
